inbox/mailsync/backends/base.py

```diff
@@ -75,12 +75,15 @@ def _run_impl(self):
         self.sync_greenlet.join()
 
         if self.sync_greenlet.successful():
-            return self._cleanup()
+            self._cleanup()
+            self.log.info(
+                "mail sync finished successfully", provider=self.provider_name
+            )
```

Comment:
Added this log here, since otherwise it was not clear that cleanup was not running